asf_core_data/pipeline/data_joining/merge_proc_datasets.py

```diff
@@ -100,14 +103,26 @@ def add_mcs_installations_data(
         epc_mcs_df["commission_date"],
         epc_mcs_df["HP_INSTALL_DATE"],
     )
+    epc_mcs_df.drop(columns="commission_date", inplace=True)
 
     # Update installation tag (will only affect MCS-only records)
     epc_mcs_df["HP_INSTALLED"] = np.where(
         epc_mcs_df["MCS_AVAILABLE"], True, epc_mcs_df["HP_INSTALLED"]
     )
 
+    # Update TECH_TYPE and standardise HP types (will only affect MCS-only records)
     epc_mcs_df = standardise_hp_type(epc_mcs_df)
 
+    # Update HEATING_SYSTEM (will only affect MCS-only records)
+    epc_mcs_df["HEATING_SYSTEM"] = np.where(
+        epc_mcs_df["MCS_AVAILABLE"], "heat pump", epc_mcs_df["HEATING_SYSTEM"]
+    )
+
+    # Update HEATING_FUEL (will only affect MCS-only records)
+    epc_mcs_df["HEATING_FUEL"] = np.where(
+        epc_mcs_df["MCS_AVAILABLE"], "electric", epc_mcs_df["HEATING_FUEL"]
+    )
+
```

Comment:
we still have to add this here @athrado. The changes to` feature_engineering.py` deal with EPC records (both match and non-matched to MCS) and here we deal with MCS records not matched to EPC.